app/server/src/routes/projectRoutes.ts

```diff
@@ -24,7 +24,7 @@ export default {
         app.post('/project/:projectId/rename',
             authCheck,
             controller.renameProject);
-        app.post(`/project/:projectId/sample/:sampleHash/delete`,
+        app.patch(`/project/:projectId/sample/:sampleHash`,
```

Comment:
I wonder if we should still have the delete route, since the fact that it's going to delete a hash/filename combo isn't obvious from the fact that it's a hash..?

Reply:
couldnt add body (need to pass file name). left as PATCH but added /delete back in to end